libs/components/lists/src/assets/locales/resources_en_US.json

```
@@ -99,6 +103,10 @@
     "_description": "Generic label for the button to move a repeater item to the top of the list. Used when a human-readable name is not provided.",
     "message": "Move to top"
   },
+  "skyux_sort_aria_label_descriptor": {
+    "_description": "Default aria label for a sort button when a descriptor is given",
```

```suggestion
    "_description": "Default ARIA label for a sort button when a descriptor is given",
```
Also, same question as for `skyux_filter_aria_label_descriptor` about whether the "when a descriptor is given" is accurate.

---

See my comment on filter.